soh/include/message_data_textbox_types.h

```diff
@@ -0,0 +1,25 @@
+#ifndef MESSAGE_DATA_TEXTBOX_TYPES_H
```

Comment:
I didn't include it in there because message_data_static.h also isn't included in CMakeLists.txt. I wasn't sure why, so I decided to follow the pattern. I can include message_data_fmt.h, message_data_static.h and message_data_textbox_types.h to CMakeLists.txt if we want.